api_v3/services/LiveStreamService.php

```diff
@@ -649,6 +684,18 @@ public function getDetailsAction($id)
 		return $liveStreamDetails;
 	}
 
+	protected function getNumberOfViewers($entryId)
+	{
+		$cache = kCacheManager::getSingleLayerCache(kCacheManager::CACHE_TYPE_PLAYS_VIEWS);
+		if (!$cache)
+		{
+			throw new kUploadTokenException("Cache instance required for Live Viewers Caching Could not initiated", kCoreException::PLAYS_VIEWS_CACHE_NOT_INITIALIZED);
+		}
+		$numberOfViewers = $cache->get("plays_views_" . $entryId);
```

Comment:
why you are using plays_views key? you should have you own format of a key for the live viewers like "current_viewers_{entry_id}